packages/near-membrane-node/src/node-realm.ts

```diff
@@ -31,6 +32,8 @@ export default function createVirtualEnvironment(
         redConnector,
         distortionCallback,
     });
+    env.link('globalThis');
+    linkIntrinsics(env, blueGlobalThis);
```

Comment:
Node doesn't need/have unforgeable linkage, only `globalThis.globalThis` linkage and intrinsics reflection.